packages/rgbpp/tests/shared/file.ts

```diff
@@ -0,0 +1,24 @@
+import { existsSync, mkdirSync, writeFileSync } from 'node:fs';
+import { resolve, join } from 'node:path';
+
+export function saveJson(props: { filename: string; json: object; path?: string }) {
```

Comment:
I cannot find any usage of this function

Reply:
It was used during test mocking, I feel like it's still needed in the future.

Reply:
If it is needed, adding it again will be more reasonable.

Reply:
Updated at 0f8f523